test/tests/bai2.rb

```diff
@@ -13,6 +13,10 @@ def setup
     @eod_no_as_of_time = Bai2::BaiFile.parse(File.expand_path('../../data/eod_without_as_of_time.bai2', __FILE__))
     @eod_with_slash_in_continuation = Bai2::BaiFile.parse(File.expand_path('../../data/eod_with_slash_in_text.bai2', __FILE__),
                                                           continuations_slash_delimit_end_of_line_only: true)
+    @group_trailer_with_no_number_of_accounts = Bai2::BaiFile.parse(
```

Comment:
Should this be added to `@all_files` so that it gets tested?

Reply:
ah, good catch, I actually added it but miss it in git 
updated